src/classes/window/presenter.vala

```diff
@@ -272,6 +272,17 @@ namespace pdfpc.Window {
             this.button_press_event.connect(this.presentation_controller.button_press);
             this.scroll_event.connect(this.presentation_controller.scroll);
 
+            // resize the bottom text based on the window height
+            // (see http://stackoverflow.com/a/35237445/730138)
+            var bottom_text_css_provider = new Gtk.CssProvider();
+            Gtk.StyleContext.add_provider_for_screen(Gdk.Screen.get_default(),
+                bottom_text_css_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+
+            const string bottom_text_css_template = ".bottomText { font-size: %fpx; }";
+            var target_size_height = (double) this.screen_geometry.height / 400.0 * 12.0;
+            var bottom_css = bottom_text_css_template.printf(target_size_height);
```

Comment:
This will create invalid CSS if your locale does not use dots as decimal delimeter. e.g. I get `.bottomText { font-size: 32,400000px; }` with a german locale.

Better use
```
const string bottom_text_css_template = ".bottomText { font-size: %fpx; }";
var target_size_height = (int) ((double)this.screen_geometry.height / 400.0 * 12.0);
```

something like this.